File: Sources/gestion-culturelle-back/src/main/java/com/afpa/cda/dto/PanierDto.java
package com.afpa.cda.dto;

import java.util.Date;
import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Builder
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class PanierDto {
	private int id;
	private Date dateValidation;
	private List<ArticleDto> listArticles;
	
	private double total;
}

File: Sources/gestion-culturelle-front/project/src/app/model/panierDto.ts
import { ArticleDto } from './articleDto';

export class PanierDto {
    id: number;
    dateValidation: Date;
    listArticles: ArticleDto;
    total: number;

    constructor(id?: number,
        dateValidation?: Date,
        listArticles?: ArticleDto,
        total?: number

    ) {
        this.id = id;
        this.dateValidation = dateValidation;
        this.listArticles = listArticles;
        this.total = total;

    }
}


